jitpack.yml

```diff
@@ -1,8 +0,0 @@
-
-before_install:
-   - yes | $ANDROID_HOME/tools/bin/sdkmanager "platforms;android-27"
-   - yes | $ANDROID_HOME/tools/bin/sdkmanager "build-tools;27.0.3"
-install:
-   - export TMPDIR=`dirname $(mktemp)`
-   - echo "Changing into the android folder of the Reanimated module"
-   - pushd android && chmod +x gradlew && ./gradlew --stacktrace clean -Pgroup=com.github.wordpress-mobile.react-native-reanimated -Pversion=$VERSION install && popd
```

Comment:
Why is `jitpack.yml` unnecessary in this repo? Is it doing work that is no longer needed for the publishing?

Reply:
It's not necessary because Jitpack is able to figure out how to publish this library.